README.md

```diff
@@ -105,3 +105,15 @@ To rebuild only the `moveit_drake` package:
 rm -rf build/moveit_drake install/moveit_drake
 colcon build --packages-select moveit_drake
 ```
+
+## Known issues
+
+### .stl support
+
+Unfortunately, Drake does not support `.stl` files (11/28/2024, see [drake#19408](https://github.com/RobotLocomotion/drake/issues/19408)). We're working around this by replacing the `.stl` files in the urdf string
+with `.obj` files in the plugin implementations. Make sure that the moveit config you're using contains the relevant `.stl` files. If it doesn't,
+take a look into the scripts/ directory. We've provided a simple python script to add additional `.obj` files for given `.stl` files. Usage:
+
+```
+./scripts/convert_stl_to_obj.py /PATH/TO/YOUR/MESH/DIR
+```
```

Comment:
Do you need to instruct people to rebuild the package after running the script? I assume the OBJ files won't make it into install space otherwise.